specification/storage/resource-manager/Microsoft.Storage/stable/2021-02-01/storage.json

```diff
@@ -2104,6 +2104,21 @@
       ],
       "description": "KeyPolicy assigned to the storage account."
     },
+    "KeyCreationTime": {
```

Comment:
Should K be lowercase (keyCreationTime)?

Reply:
This is a swagger definition rather than a property name.
(usage: line 2995,
**"keyCreationTime": {
          "readOnly": true,
          "$ref": "#/definitions/KeyCreationTime",
          "description": "Storage account keys creation time."
        }**,)
And it's a naming convention that we should use uppercase on a definition.

Reply:
Oh, ok, my mistake.